cmd/tools/rest/href_builder.go

```diff
@@ -76,6 +85,26 @@ func (b *HrefBuilder) Build() string {
 
 	href.WriteString("?return_records=true")
 
+	if len(strings.Join(b.fields, ",")) > URLMaxLimit {
+		fmt.Printf("converting to * due to URL max limit")
+		b.fields = []string{"*"}
+	}
+
+	if len(b.hiddenFields) > 0 {
```

Comment:
If hidden fields are added below the URLMaxLimit check, they may cause the URL to exceed the max. Hidden fields need to be part of the max check

Reply:
I was thinking that, if hiddenFields would be prior then if it exceed the limit, then it will pass `*` and no hidden fields would be coming in that response. 
But yeah, if it was in later then url rest limit 414 error will come and whole rest call would be failed.